rest/handler.go

```diff
@@ -134,6 +135,20 @@ func makeOfflineHandler(server *ServerContext, privs handlerPrivs, accessPermiss
 	})
 }
 
+// Create a http.Handler that will run a handler with the given method. It also takes in a callback function which when
+// given the endpoint payload returns an auth scope. Currently, used for create db where we need to read bucket from the
```

Comment:
I think you can remove the 'Currently' comment - it's straightforward to find usage.

I think it's probably clearer to just call this makeAuthScopeHandler as well.